manifests/slave.pp

```diff
@@ -77,6 +80,7 @@
   $slave_name               = undef,
   $description              = undef,
   $masterurl                = undef,
+  $tunnel                   = undef,
```

Comment:
Please validate the input (I realize this module is inconsistent about input validation but we should do it for all new params).